src/components/issues/IssuesPage.js

```diff
@@ -68,11 +73,29 @@ const Wrapper = styled.div`
   margin: 70px auto 0;
 `;
 
+const Loader = styled.div`
```

Comment:
you should probably extract this to a common component (components/common/Loader)